src/components/SystemMessageNotification/SystemMessageNotif.tsx

```diff
@@ -0,0 +1,40 @@
+import React, {useState} from 'react';
+import {Alert} from 'react-bootstrap';
+import styles from './SystemMessageNotification.module.scss';
+import {SystemMessageNotification} from '../../types/SystemMessageNotification';
+
+
+export function getAlertTypeClass(type: SystemMessageNotification) {
+  if (type === 'Info') {
+    return styles['info'];
+  }
+  else if (type === 'Warning') {
+  return styles['warning'];
+  }
+  else
+    return styles['notice'];
+
+}
+
+interface Props {
+  onClick?: () => void;
+  id: number;
+  type: SystemMessageNotification;
+
+  description: string;
```

Comment:
take these props out and replace it with the message object. Fix the caller and the definition
```suggestion
  message: SystemMessage
```